README.md

```diff
@@ -1,28 +1,65 @@
 # XMTP-MEMO-JS
 
-**XMTP client SDK for Pre-Registration messaging**
+![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-red)
 
-`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
-Build with `xmtp-memo-js` to send 'messages' to a blockchain wallet address which does not have published encryption keys on the network. `xmtp-memo-js` supports use-cases where content needs to be sent to a account which has not registered yet.
+## XMTP client SDK for pre-registration messaging
 
-**What is a memo?**
+`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
 
-Memo's are similar to XMTP Messages however they differ substantively.
+With XMTP, a blockchain account address must be registered on the XMTP network to send and receive end-to-end encrypted messages. Build your app with `xmtp-memo-js` to enable a registered address to start communicating with an unregistered address by sending a memo instead of a message.
```

Comment:
```suggestion
With XMTP, a blockchain account must be registered on the XMTP network to send and receive end-to-end encrypted messages. Build your app with `xmtp-memo-js` to enable a registered account to start communicating with an unregistered account by sending a memo instead of a message.
```